index.html

```diff
@@ -1,2 +1,13 @@
-<h1>Krystian Pyszniak</h1>
-sdada
\ No newline at end of file
+<!doctype html>
+<html lang="pl">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>CV - Krystian Pyszniak</title>
```

Comment:
Why this title?

Reply:
Because Im Krystian Pyszniak and fuck off